script.js

```diff
@@ -199,3 +199,16 @@ document.querySelectorAll('.openPopupWindow').forEach((item) => {
   });
 });
 
+const form = document.querySelector('form');
+const error = document.querySelector('.error-message');
+const email = document.getElementById('email');
```

Comment:
- [ ] The email field of your form is accepting incorrect email input. I imputed an incomplete email `se@f` and this was submitted successfully. Kindly ensure that your email field is properly validated against the wrong email format